package.json

```diff
@@ -17,7 +17,8 @@
   "license": "Apache-2.0",
   "private": true,
   "scripts": {
-    "test": "gulp ci"
+    "test": "gulp ci",
+    "bootstrap": "npm install -g bower gulp jison"
```

Comment:
We need a readme with a set of terminal steps from "nothing" state to running the app.